kaldb/src/main/resources/KaldbLayout.json

```diff
@@ -22,38 +22,40 @@
     "$resolver": "logger",
     "field": "name"
   },
-  "labels": {
+  "traceId": {
     "$resolver": "mdc",
-    "flatten": true,
+    "key": "traceId",
     "stringified": true
   },
-  "tags": {
-    "$resolver": "ndc"
+  "spanId": {
+    "$resolver": "mdc",
+    "key": "spanId",
+    "stringified": true
   },
-  "error.type": {
+  "error_type": {
     "$resolver": "exception",
     "field": "className"
   },
-  "error.message": {
+  "error_message": {
     "$resolver": "exception",
     "field": "message"
   },
-  "error.stack_trace": {
+  "error_stack_trace": {
```

Comment:
Instead of `error_stack_trace` would `error:stack_trace` work?

Reply:
`:` is a lucene delimiter for fieldname:fieldvalue query in lucene.

So I don't think `error:stack_trace` will work

Reply:
Sounds good!